Soccar [Game-Server]/src/nl/soccar/gameserver/util/MapUtilities.java

```diff
@@ -120,43 +126,8 @@ private static void addNorthAndSouthWalls(GameEngine engine, float mapWidth, flo
     }
 
     /**
-     * Method that adds the obstacle drawables to the map that represent the
-     * nort and south walls.
-     *
-     * @param mapWidth  The width of the map.
-     * @param mapHeight The height of the map.
-     */
-    private static void addCornerWalls(GameEngine engine, float mapWidth, float mapHeight) {
-        World world = engine.getWorld();
-
-        ObstaclePhysics northWestWall = new ObstacleBuilder(world)
-                .x(0).y(mapHeight).degree(45)
-                .width(CORNER_SIZE).height(CORNER_SIZE)
-                .type(ObstacleType.WALL).build();
-
-        ObstaclePhysics northEastWall = new ObstacleBuilder(world)
-                .x(mapWidth).y(mapHeight).degree(45)
-                .width(CORNER_SIZE).height(CORNER_SIZE)
-                .type(ObstacleType.WALL).build();
-
-        ObstaclePhysics southWestWall = new ObstacleBuilder(world)
-                .x(0).y(0).degree(45)
-                .width(CORNER_SIZE).height(CORNER_SIZE)
-                .type(ObstacleType.WALL).build();
-
-        ObstaclePhysics southEastWall = new ObstacleBuilder(world)
-                .x(mapWidth).y(0).degree(45)
-                .width(CORNER_SIZE).height(CORNER_SIZE)
-                .type(ObstacleType.WALL).build();
-
-        engine.addWorldObject(northWestWall);
-        engine.addWorldObject(northEastWall);
-        engine.addWorldObject(southWestWall);
-        engine.addWorldObject(southEastWall);
-    }
-
-    /**
-     * An ObstacleBuilder builds an obstacle for usage with the UI. It combines a model, physics-model and ui object and builds them implicitely.
+     * An ObstacleBuilder builds an obstacle for usage with the UI. It combines
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "addCornerWalls" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUnusedPrivateMethod)